ouroboros-network/test/messages.cddl

```diff
@@ -1,27 +1,17 @@
-; The only purpose of allMessages is to have an entry point for testing all parts of the specification.
+; The purpose of allMessages is to have an entry point for testing all parts of the specification.
 ; ! allMessages is NOT a valid wire format !
 
-allMessages =
-      [0,chainSyncMessage]
+allMessages
+    = [0,chainSyncMessage]
     / [1,reqRespMessage]
     / [2,pingPongMessage]
```

Comment:
Testing for ping pong seems to be only partially removed. Is there a reason for it not being completely removed from the spec and `CLDL.hs`?